### PR TITLE
Feature/create skywire cli pk command 238

### PR DESCRIPTION
**Pull request by ayuryshev**
Changes:

1. Create command `skywire-cli pk`
    
E.g.    

```bash
$ ./skywire-cli --rpc $SW_NODE_B:3435 pk
# 0297d4b098776b16a80ee43cdb66c4aaa09bf72ba2215bbfdc94bd96437b61759b
$ echo $(./skywire-cli pk)
# 02eca7c44ceced3249fc8f0610034d354f5ceba629377e7034e709b5d6823ed399
```

Additional changes:
    
1. Removed incorrect Example for client
2. Fixed typo "TransportManger" in pkg/transport/manager_test.go    
(  as discussed with @evanlinjin )

### CODE REVIEW

**Comment by ayuryshev** [comment]
Closes: #238

File: cmd/skywire-cli/commands/pk.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+var pkCmd = &cobra.Command{
+	Use:   "pk",
+	Short: "get public key of node",
+	Run: func(_ *cobra.Command, _ []string) {
+
+		client := rpcClient()
+		summary, err := client.Summary()
+		if err != nil {
+			log.Fatal("Failed to connect", err)
```
Comment (evanlinjin) [inline cmd/skywire-cli/commands/pk.go]: Have a colon `:` at the end of `connect`. Thanks!

**Review by evanlinjin — APPROVED** [review]
Just fix this up and it's ready to merge!

File: cmd/skywire-cli/commands/pk.go (hunk 1 above)
Comment (ayuryshev) [reply cmd/skywire-cli/commands/pk.go]: Done!

**Pull request merged**